src/ebusd/mqtthandler.cpp

```diff
@@ -890,6 +905,16 @@ void MqttHandler::publishMessage(const Message* message, ostringstream* updates,
     }
     if (json) {
       *updates << "{";
+
```

Comment:
this should become a more generic setting, e.g. extend the json setting by something like 'withCircuitName'